contribs/gmf/src/dropfile/directive.js

```diff
@@ -0,0 +1,119 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2019-2020 Camptocamp SA
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of
+// this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to
+// use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+// the Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+// COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+// IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+// CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import angular from 'angular';
+import {MessageType} from 'ngeo/message/Message.js';
+import './fileAlert.scss';
+
+/**
+ * @ngInject
+ * @param {import("ngeo/message/Notification.js").MessageNotification} ngeoNotification Ngeo notification
+ * @param {import("gmf/datasource/ExternalDataSourcesManager.js").ExternalDatSourcesManager} gmfExternalDataSourcesManager The manager of external datasources.
+ * @param {angular.gettext.gettextCatalog} gettextCatalog Gettext catalog.
+ * @return {angular.IDirective} The Directive Definition Object.
+ * @ngdoc directive
+ * @ngname gmfFileDropZone
+ */
+const fileDrop = function (gmfExternalDataSourcesManager, ngeoNotification, gettextCatalog) {
```

Comment:
Can you add some description to know what this fileDrop function do (=> it adds an overlay... you can disable it with...)  ?